solidity/contracts/isms/hook/ArbL2ToL1Ism.sol

```diff
@@ -45,15 +46,12 @@ contract ArbL2ToL1Ism is
 
     // ============ Constructor ============
 
-    constructor(
-        address _bridge,
-        address _outbox
-    ) CrossChainEnabledArbitrumL1(_bridge) {
+    constructor(address _bridge) CrossChainEnabledArbitrumL1(_bridge) {
         require(
             Address.isContract(_bridge),
             "ArbL2ToL1Ism: invalid Arbitrum Bridge"
         );
-        arbOutbox = IOutbox(_outbox);
+        arbOutbox = IOutbox(IBridge(_bridge).activeOutbox());
```

Comment:
can this change?

Reply:
no, it's in the initializer here: https://github.com/OffchainLabs/nitro-contracts/blob/69399a22fdc3102120d5ac0737d0830580218aeb/src/bridge/Bridge.sol#L25